CHANGELOG.md

```diff
@@ -38,6 +38,8 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 ## Unreleased
 
+- **AWS Scalers**: Add support for override the AWS endpoint url. ([#3337](https://github.com/kedacore/keda/issues/3337))
+
```

Comment:
Could you move this under `### New`and short it following the [style guide](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog)?

Reply:
> short it following the [style guide](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog)

Done
https://github.com/kedacore/keda/pull/3779/commits/db3de036a8493c229ac1704b17d0b2893b742b21